src/Sentry/SentryOptions.cs

```diff
@@ -77,6 +77,16 @@ public bool IsGlobalModeEnabled
         set => _isGlobalModeEnabled = value;
     }
 #endif
+    private NdkHandlerStrategy _handlerStrategy = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT;
```

Comment:
Why do we need the private backing field?

Couldn't this be a 1 liner?

```csharp
public NdkHandlerStrategy HandlerStrategy {get;set;} = NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT;
```

Reply:
absolutely, I don't know why I did it this way. 
I think I was going for another approach at first